DxDispatch/src/model/ImageReaderWriter.cpp

```diff
@@ -0,0 +1,271 @@
+#include "pch.h"
+#include "StdSupport.h"
+#include "ImageReaderWriter.h"
+
+#ifndef WIN32
+std::vector<std::byte> ReadTensorFromImage(
+    std::filesystem::path srcPath,
+    const ImageTensorInfo& dstTensorInfo)
+{
+    throw std::runtime_error("ReadTensorFromImage is not implemented on this platform.");
+}
+
+void WriteTensorToImage(
+    std::filesystem::path dstPath,
+    std::span<const std::byte> srcData,
+    const ImageTensorInfo& srcInfo)
+{
+    throw std::runtime_error("WriteTensorToImage is not implemented on this platform.");
+}
+
+#else // Win32
+
+#include <wincodec.h>
+
+using Microsoft::WRL::ComPtr;
+
+size_t GetDataTypeSize(DML_TENSOR_DATA_TYPE dataType)
+{
+    switch (dataType)
+    {
+        case DML_TENSOR_DATA_TYPE_FLOAT32: return sizeof(float);
+        case DML_TENSOR_DATA_TYPE_FLOAT16: return sizeof(uint16_t);
+        case DML_TENSOR_DATA_TYPE_INT8: return sizeof(int8_t);
```

Comment:
Unsigned `DML_TENSOR_DATA_TYPE_UINT8`? I've seen UINT8 as graph inputs to some models (which is supported by `DML_CONVOLUTION_INTEGER_OPERATOR_DESC`), but I haven't seen INT8.